src/error_messages/warning_messages.h

```diff
@@ -71,6 +71,8 @@
 #define FIM_WHODATA_POLICY_CHANGE_CHECKER       "(6952): Audit policy change detected. Switching directories to realtime."
 #define FIM_WHODATA_POLICY_CHANGE_CHANNEL       "(6953): Event 4719 received due to changes in audit policy. Switching directories to realtime."
 #define FIM_EMPTY_CHANGED_ATTRIBUTES            "(6954): Entry '%s' does not have any modified fields. No event will be generated."
+#define FIM_AUDITPOL_ATTEMPT_FAIL               "(6955): Auditpol command failed, attempt number: %d"
```

Comment:
These messages should be write in the error_messages.h file, in the same folder.

Reply:
Addressed with https://github.com/wazuh/wazuh/pull/21455/commits/27c2e8f3c801ad04126261eb11f62862b9623a07